felix/wireguard/wireguard.go

```diff
@@ -270,6 +270,15 @@ func (w *Wireguard) OnIfaceStateChanged(ifaceName string, state ifacemonitor.Sta
 	w.routetable.OnIfaceStateChanged(ifaceName, state)
 }
 
+// ReportZeroKey uses the configured statusCallback to report an empty WG key.
+// This is done to bootstrap connectivity to wireguard-enabled peers such as Typha
+// who would otherwise attempt to send us wireguard traffic before we are able to receive it
+func (w *Wireguard) ReportZeroKey() error {
```

Comment:
I need to make this smarter. Currently the node's wireguard key is _always_ set to zero on-felix-startup